src/consensus/params.h

```diff
@@ -95,6 +95,23 @@ struct Params {
 
     bool fSimplifiedRewards;
     bool fShortEarlyCoinbase;
+
+    /** Auxpow parameters */
+    int32_t nAuxpowChainId;
+    bool fStrictChainId;
```

Comment:
Those 2 new variables are not used: is it expected?

Reply:
Yup, I'm trying to break #1974 down into bite-size chunks, or at least reduce the number of files it touches.

Reply:
We should maybe first do a PR importing all new auxpow files and then another PR implementing auxpow logic into Dogecoin

Reply:
Kind of trying a few approaches in parallel. For parameters, they're critical to get right, so worth raising by themselves.

It's tricky to just add the new files (because to make some of them compile, existing files are modified). However can likely add AuxPoW block validation in one PR, and mining in a second PR, which should bring it down. You're very welcome to take a stab at splitting up 1974 though.